libraries/JGUZDV.JobHost/src/JGUZDV.JobHost/src/JobHost.cs

```diff
@@ -33,7 +33,9 @@ public static IHostBuilder CreateJobHostBuilder(string[] args, Action<WindowsSer
                                     x.AwaitApplicationStarted = true;
                                 });
 
-                                if (ctx.HostingEnvironment.IsDevelopment())
+                                var allowDebugging = ctx.Configuration.GetValue<bool?>($"{Constants.DefaultConfigSection}:AllowDebugging") ?? false;
```

Comment:
Mein Kommentar zu "deaktivieren" war hier ernst gemeint.
Ich halte opt-out an dieser Stelle für weitaus nützlicher, als opt-in.

"DisableDevelopmentJobSelection"